docs/vNext/MonitorsAndObservers.md

```diff
@@ -94,11 +94,12 @@ All data stored on Prometheus may be queried by Caliper using the Prometheus que
 The prometheus monitoring module options comprise:
 - interval: monitor update interval
 - url: The Prometheus URL, used for direct queries
-- metrics: The queries to be run for inclusion within the Caliper report, comprised of to keys: `ignore` and `include`. 
-  - `ignore` a string array that is used as a blacklist for report results. Any results where the component label matches an item in the list, will *not* be included in a generated report.
-  - `include` a series of blocks that describe the queries that are to be run at the end of each Caliper test.
+- metrics: The queries to be run for inclusion within the Caliper report, comprised of to keys: `include` and `queries`. 
+  - `include` a string array that is used to determine metric inclusion through javascript regex. Any results where the component label matches an item in the list via regex, will be included in a generated report.
```

Comment:
Sorry dumb question here, but what is a component label ?

Reply:
no such thing as a bad question! So a "component label" is basically anything "named" that prometheus knows about, such as a docker container name. The context jumps around a little depending on the specific queries you are running and the scrape targets available but in the general case a prometheus query will come back with a set of ({componentName: value}, {componentName: value}, ...) and we can then filter on the things we are actually interested in